test/powershell/Modules/Microsoft.PowerShell.Management/Copy-Item.Tests.ps1

```diff
@@ -274,6 +279,15 @@ Describe "Validate Copy-Item Remotely" -Tags "CI" {
                 $copiedFilePath | Should -Exist
             }
         }
+
+        It "Copy-Item Locally and Retain Attributes" {
+            $srcDir = GetSourceFolderPath
+            $destDir = GetDestinationFolderPath
+            $hiddenFolder = New-Item -Path (Join-Path -Path $srcDir -ChildPath "HiddenFolder") -ItemType Directory
+            $hiddenFolder.Attributes += "Hidden"
```

Comment:
Hidden attribute is not cross-platform. I suggest to use another attribute for the test.

Reply:
I have updated the test to use the "System" attribute.